src/routes/projects/+page.svelte

```diff
@@ -7,3 +8,10 @@
 <article class="prose max-w-full dark:prose-invert">
     <FilterPagePanel {filters} cardsInfo={projects} />
 </article>
+
+<section>
+    <h1 class="mb-4 w-2/5 text-3xl text-csi-black md:text-4xl dark:text-csi-white">FAQs</h1>
+    <Accordion title="Lorem ipsum dolor sit amet?">
+        <span> Lorem ipsum dolor sit amet adipiscing elit consectitur </span>
+    </Accordion>
```

Comment:
```suggestion
    <div class="flex flex-col gap-6">
        <Accordion title="Lorem ipsum dolor sit amet?">
            <span> Lorem ipsum dolor sit amet adipiscing elit consectitur </span>
        </Accordion>
    </div>
```
Considering there will be an accordion for each FAQ, might be worth putting it inside a `<div>` in advance.